tools/umi_tools/test/main.nf

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env nextflow
+
+// Define DSL2
+nextflow.preview.dsl=2
+
+// Log
+log.info ("Starting test for umi_tools dedup...")
+
+/*------------------------------------------------------------------------------------*/
+/* Define params
+--------------------------------------------------------------------------------------*/
+
+params.umitools_dedup_args = '--umi-separator=":"'
+params.verbose = true
+
+/*------------------------------------------------------------------------------------*/
+/* Module inclusions
+--------------------------------------------------------------------------------------*/
+
+include umitools_dedup from '../main.nf'
+
+/*------------------------------------------------------------------------------------*/
+/* Define input channels
+--------------------------------------------------------------------------------------*/
+
+// Meta data
+testMetaDataBam = [
+  ['sample1', "$baseDir/input/sample1.bam"],
```

Comment:
2-tab spacing mixed with 4-tab spacing 😅

Reply:
tragic! **hangs head in shame**